mlir/utils/buildbot/mlir-rocm-mi200/Dockerfile

```diff
@@ -15,25 +15,18 @@
 FROM rocm/mlir:rocm5.1-latest
 
 RUN apt-get update; \
-    apt-get install -y dumb-init; \
+    apt-get install -y dumb-init cmake; \
     update-alternatives --install /usr/bin/clang clang /usr/bin/clang-10 100 ;\
     update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-10 100 ;\
     update-alternatives --install /usr/bin/lld lld /usr/bin/lld-10 100
 
 # LTS releases often bundle obsolete pip versions that cannot access newest
 # Linux binary wheels.
-RUN python3 -m pip install --upgrade pip
-
-RUN apt-add-repository ppa:deadsnakes/ppa && apt-get update \
-     && apt-get install -y --no-install-recommends \
-            python3.8 python3.8-dev python3.8-venv \
-     && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 10 \
```

Comment:
I think we might still need the `update-alternatives` call to set system `/usr/bin/python3`

Reply:
Already done in mlir/utils/jenkins/Dockerfile, which has a similar section.